portfolio/src/main/webapp/style.css

```diff
@@ -7,3 +7,54 @@
 #greeting-container {
   margin-top: 20px;
 }
+
+body {
+  background-color: #fbfcfc;
+}
+
+
+/* Styling the header */
+
+.header {
+  overflow: hidden;
+  background-color: #fbfcfc;
+  padding: 20px 10px;
+  margin-left: auto;
+  margin-right:auto;
+  width: 750px;
+}
+
+/* Stylize image in header as a profile photo */
+
+.header img {
+  float: left;
+  border-radius: 70%;
+  border: 2px solid black;
+  height: 150px;
+  width: 150px;
```

Comment:
nit: if you omit one of these, it'll preserve the aspect ratio of the image (assuming it's not square)